src/cljc/athens/common_events.cljc

```diff
@@ -441,6 +441,64 @@
                      :target-uid  target-uid
                      :drag-target drag-target}}))
 
+(defn build-drop-same-event
+  "Builds `:datascript/drop-same` event with:
+  - `source-uid` : uid of the source block
```

Comment:
What is the role of `drag-target` and what is it?

Reply:
`target-uid` is the uid of block to which some block(s) are being added but these blocks can be added above or below the target block. `drag-target` represents this situation of where the blocks are being added on the target block.

While rethinking drop events implementation I found that this is not the correct information to information to send, we can send better crisp information.